### PR TITLE
refactor: use fld and cld instead of floor and ceil

### PR DESCRIPTION
**Pull request by teresy**
I noticed these cases where it looks like `fld` and `cld` can be used, respectively.

### CODE REVIEW

File: stdlib/DelimitedFiles/test/runtests.jl [hunk 1]
```diff
@@ -217,7 +217,7 @@ end
             "Yugoslavia (Cyrillic)", "Djordje Balasevic", "Југославија", "Ђорђе Балашевић",
             "Yugoslavia (Latin)", "Djordje Balasevic", "Jugoslavija", "Đorđe Balašević"]
 
-        i18n_arr = permutedims(reshape(i18n_data, 4, Int(floor(length(i18n_data)/4))), [2, 1])
+        i18n_arr = permutedims(reshape(i18n_data, 4, Int(fld(length(i18n_data),4))), [2, 1])
```
Comment (StefanKarpinski) [inline stdlib/DelimitedFiles/test/runtests.jl]: If we keep this one, you can remove the call to `Int` since the result will be of type `Int` already.

**Comment by StefanKarpinski** [comment]
Somewhat counterintuitively, integer division is much slower than floating-point division. Moreover, when a value is already in a floating-point register, doing an integer division requires moving it to a different register, then doing the operation and then moving it back, which is even slower.

**Comment by teresy** [comment]
Right, so these changes may help along those lines (which I picked up here: https://docs.julialang.org/en/v1/manual/performance-tips/index.html#Tweaks-1). I find @timholy 's snippet interesting though since it appears that outside tight loops there may be a downside. Short of benchmarking the changes, I'm leaving this to your discretion.

**Comment by dpsanders** [comment]
Apart from anything else, the names fld and cld are much less clear.

**Comment by StefanKarpinski** [comment]
> Short of benchmarking the changes, I'm leaving this to your discretion.

Benchmarks showing that this is not slower would be exactly what is needed to merge this PR...

**Comment by teresy** [comment]
Ok sounds good, I'll update once I have a chance to look into benchmarks.

**Comment by StefanKarpinski** [comment]
Don't worry about the tests—that one's good. It's the upper triangular matrix log that matters.

**Pull request closed without merging**